staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go

```diff
@@ -139,6 +142,9 @@ func newStore(c *clientv3.Client, codec runtime.Codec, newFunc, newListFunc func
 	w.getCurrentStorageRV = func(ctx context.Context) (uint64, error) {
 		return storage.GetCurrentResourceVersionFromStorage(ctx, s, newListFunc, resourcePrefix, w.objectType)
 	}
+	if utilfeature.DefaultFeatureGate.Enabled(features.ConsistentListFromCache) {
```

Comment:
let's use also WatchList feature [either of those should activate it]

Reply:
Done, but you need to use that in other places. I only used `Supports` for `ConsistentListFromCache` active.

Reply:
I think that all other places are consistent-list specific. But we want to ensure that we enable checking on both features.